ddsp/training/train_util.py

```diff
@@ -140,6 +140,13 @@ def format_for_tensorboard(line):
     tf.summary.text(name='gin/' + base_name, data=text_tensor, step=step)
     summary_writer.flush()
 
+def report_metric_to_ai_platform(loss_report, step):
```

Comment:
nit, I would make the loss_report arg `metric_value` and add a kwarg at the end `tag='Loss'` to make this function more general.

I would also call it `report_metric_to_hypertune()` as I think you can use the package outside ai-platform as well